jobsdb/jobsdb_export.go

```diff
@@ -112,9 +113,13 @@ func (jd *HandleT) GetNonMigratedAndMarkMigrating(count int) []*JobT {
 	jd.assertError(err)
 
 	for ds, updatedStatesByCustomer := range updatedStatesByDS {
+		allUpdatedStates := make([]string, 0)
```

Comment:
Maybe we should consider using https://github.com/deckarep/golang-set in such cases. I was also considering using this library for dedup as well. I am not sure of the performance benefits though.